lib/streamlit/watcher/__init__.py

```diff
@@ -11,3 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from .local_sources_watcher import LocalSourcesWatcher as LocalSourcesWatcher
+from .path_watcher import (
+    report_watchdog_availability as report_watchdog_availability,
+    watch_file as watch_file,
```

Comment:
Is there a reason why to use import as?

Reply:
`mypy` (or at least the way we have `mypy` configured) requires you to explicitly name functions that you're reexporting

Reply:
mypy (or at least the way we have mypy configured) requires you to explicitly name functions that you're reexporting